bindings/python/src/pipeline/datatype/ImgFrameBindings.cpp

```
@@ -172,10 +188,29 @@ void bind_imgframe(pybind11::module& m, void* pCallstack) {
         .def("addFlipHorizontal", &ImgTransformation::addFlipHorizontal, DOC(dai, ImgTransformation, addFlipHorizontal))
         .def("addRotation", &ImgTransformation::addRotation, py::arg("angle"), py::arg("rotationPoint"), DOC(dai, ImgTransformation, addRotation))
         .def("addScale", &ImgTransformation::addScale, py::arg("scaleX"), py::arg("scaleY"), DOC(dai, ImgTransformation, addScale))
+        .def("addSrcCrops", &ImgTransformation::addSrcCrops, py::arg("crops"), DOC(dai, ImgTransformation, addSrcCrops))
         .def("remapPointTo", &ImgTransformation::remapPointTo, py::arg("to"), py::arg("point"), DOC(dai, ImgTransformation, remapPointTo))
-        .def("remapPointFrom", &ImgTransformation::remapPointFrom, py::arg("to"), py::arg("point"), DOC(dai, ImgTransformation, remapPointFrom))
+        .def("remapPointFrom", &ImgTransformation::remapPointFrom, py::arg("from"), py::arg("point"), DOC(dai, ImgTransformation, remapPointFrom))
         .def("remapRectTo", &ImgTransformation::remapRectTo, py::arg("to"), py::arg("rect"), DOC(dai, ImgTransformation, remapRectTo))
-        .def("remapRectFrom", &ImgTransformation::remapRectFrom, py::arg("to"), py::arg("rect"), DOC(dai, ImgTransformation, remapRectFrom))
+        .def("remapRectFrom", &ImgTransformation::remapRectFrom, py::arg("from"), py::arg("rect"), DOC(dai, ImgTransformation, remapRectFrom))
+        .def("project3DPoint", &ImgTransformation::project3DPoint, py::arg("point"), DOC(dai, ImgTransformation, project3DPoint))
+        .def("project3DPointTo", &ImgTransformation::project3DPointTo, py::arg("to"), py::arg("point"), DOC(dai, ImgTransformation, project3DPointTo))
```

Cppcheck (2.20.0)

[error] There is an unknown macro here somewhere. Configuration is required. If DEPTHAI_NLOHMANN_DEFINE_TYPE_INTRUSIVE is a macro then please configure it.

(unknownMacro)